onnxruntime/core/providers/rocm/tunable/gemm_hipblaslt.h

```diff
@@ -204,6 +207,13 @@ Status HipBlasLtStridedBatchedGemmOp(const StridedBatchedGemmParams<T>* params)
   return HipBlasLtMatMul<T, StridedBatchedGemmParams<T>>(params, params->batch);
 };
 
+template <typename T>
+Status HipBlasLtGemmGeluOp(const GemmFastGeluParams<T>* params) {
+  TUNABLE_OP_RETURN_UNSUPPORTED_ARGUMENT_IF((std::is_same_v<T, double>), "hipBLASLt does not support double inputs");
+  TUNABLE_OP_RETURN_UNSUPPORTED_ARGUMENT_IF((nullptr != params->bias), "hipBLASLt does not support bias gelu");
```

Comment:
hipBLASLt has bias support and epilogue `HIPBLASLT_EPILOGUE_GELU_BIAS`, does it meet the need here?

Reply:
Thanks for your reminder.